de.tudarmstadt.ukp.wikipedia.api/src/main/java/de/tudarmstadt/ukp/wikipedia/api/sweble/PlainTextConverter.java

```diff
@@ -176,13 +176,19 @@ protected Object after(WtNode node, Object result)
 
 	// =========================================================================
 
+	/*
+	 * We CAN NOT allow this method being implemented here, as it will clash with
```

Comment:
I have no idea, but if you are interested, then using the "blame/annotate" feature might shed some light. The repo contains the history of JWPL since it was on Google Code - although not from the time before it was open-sourced.